src/Data/MongoDBMock/contests.js

```diff
@@ -0,0 +1,147 @@
+const contests = [
+  {
+    contestId: 'a0347677-c3c9-4edc-9d46-fed4a958fdc2',
+    contestName: 'XII Zawody im. Pana Starosty',
+    kynologiqueDepartment: 'Gdynia',
+    startDate: new Date(2022, 3, 22, 10, 0),
+    endDate: new Date(2022, 3, 23),
+    applicationOpenDate: new Date(2022, 1, 10, 10, 0),
+    applicationClosedDate: new Date(2022, 2, 20, 23, 59),
+    place: 'Gdańsk',
+    detailAdress: 'ul.Grunwaldzka 222',
+    judges: ['Alberto Makkaroni'],
+    steward: 'Wiesio Kapusta',
+    manager: 'Jagienka Krzywobroda',
+    feePLN: 90,
+    participants: ['matylda1234', 'bogdan678', 'zosia9474', 'eustachy123'],
+    obedienceClasses: {
+      0: [
+        {
+          dogName: 'Woof',
+          summaryId: '22-03-22Woof',
+        },
+        {
+          dogName: 'Nosek',
+          summaryId: '22-03-22Nosek',
+        },
+        {
+          dogName: 'Ptysio',
+          summaryId: '22-03-22Ptysio',
+        },
+      ],
+      1: [
+        {
+          dogName: 'Ollie',
+          summaryId: '22-03-22Ollie',
+        },
+        {
+          dogName: 'Kluska',
+          summaryId: '22-03-22Kluska',
+        },
+        {
+          dogName: 'Norah',
+          summaryId: '22-03-22Norah',
```

Comment:
I think the summaries are a great idea. The data wont change over time, right? After the competition is done, there is no editing it?

Reply:
exactly, what happend at competition, is never chnged later